pkg/os/disk/api.go

```diff
@@ -93,8 +93,8 @@ func (DiskAPI) ListDiskLocations() (map[uint32]shared.DiskLocation, error) {
 
 	m := make(map[uint32]shared.DiskLocation)
 	for _, v := range getDisk {
-		str := v["location"].(string)
-		num := v["number"].(float64)
+		str := v["Location"].(string)
```

Comment:
why change location --> Location?

Reply:
this change is actually not needed and I can revert it back, it's because I also changed this line above:

```
cmd := fmt.Sprintf("ConvertTo-Json @(Get-Disk | select Number, Location)")
```

Reply:
I think I can skip making this change, I already run the integration tests on #155 and they're passing with the v1 API too